app/gateway/database/postgres/accounts/get_by_id_test.go

```diff
@@ -14,22 +14,24 @@ func Test_GetByID(t *testing.T) {
 	database := databaseTest
 	accountRepository := NewAccountRepository(database)
 	testCases := []struct {
-		name    string
-		input   account.Account
-		want    account.Account
-		wanted  string
-		wantErr bool
+		name      string
+		input     account.Account
+		want      account.Account
+		runBefore func(value string) string
+		wanted    string
+		wantErr   bool
 	}{
 		{
 			name: "localizado a conta usando o ID externo (uuid), e retorna os dados da mesma ",
 			input: account.Account{
```

Comment:
acredito que seu input nos testes dessa função seria o `types.AccountID` para ficar consistente com o que a função recebe como entrada, não sendo necessário criar um atributo `wanted` para isso. Faz sentido?

Reply:
Na real tem que ser um campo externo Leandro, para quando eu for fazer o teste com um ID inexistente, eu não precisar informar os dados em um objeto de conta falso.

Reply:
Acrescentando a resposta: eu vi que seu input é usado para criar uma conta antes de rodar o teste, acho que ficaria mais claro que é esse o comportamento realizando essa inserção no runBefore, e ele continuaria retornando o ID do que foi inserido. Me diz ai o que tu acha

Reply:
Converti o tipo do wanted para accountID, veja se fica melhor assim.

Reply:
No caso de colocar no RunBefore, ao menos nos testes que eu fiz, eu teria que reproduzir o comportamento do create, no runBefore, o que levaria a duplicação do código, por isso preferi essa abordagem de deixar ali através do create.
Tem alguma dica de como eu poderia simplificar isso?